lib/process_settings/abstract_monitor.rb

```diff
@@ -21,8 +21,6 @@ def initialize(logger:)
       @static_context = {}
     end
 
-    # []
-    #
     # This is the main entry point for looking up settings on the Monitor instance.
```

Comment:
I shouldn't have the method name repeated up here for `rdoc`, right?

Reply:
I don't think so. This is a good removal